o2physics.sh

```diff
@@ -13,8 +13,17 @@ build_requires:
   - ninja
   - alibuild-recipe-tools
 source: https://github.com/AliceO2Group/O2Physics
+track_env:
```

Comment:
@singiamtel can you add it to the linter, please?

Reply:
Will do. Could you give me write access to the repo? The infra admins only had read permissions there for some reason

Reply:
done